src/ontodia/data/demo/data/elements.json

```diff
@@ -685,7 +685,7 @@
     "http://ailab.ifmo.ru/dialog/tv/schema#Router": {
         "id": "http://ailab.ifmo.ru/dialog/tv/schema#Router",
         "types": [
-            "http://www.w3.org/2002/07/owl#Class"
+            "http://www.w3.org/2002/07/owl#Thing"
```

Comment:
Why do you need to change default example? Do you need it to demo something? I expect there are enough classes in this example to showcase visualization customization.

Reply:
I wanted to experiment with the data. Returned as it was